gslib/commands/cp.py

```diff
@@ -81,194 +81,188 @@
 
-  The contents of stdin can name files, cloud URLs, and wildcards of files
-  and cloud URLs.
+  where the output of ``cat filelist`` is a list of files, cloud URLs, and
+  wildcards of files and cloud URLs.
 
-  NOTE: Shells (like bash, zsh) sometimes attempt to expand wildcards in ways
-  that can be surprising. Also, attempting to copy files whose names contain
-  wildcard characters can result in problems. For more details about these
-  issues see the section "POTENTIALLY SURPRISING BEHAVIOR WHEN USING WILDCARDS"
-  under "gsutil help wildcards".
+  NOTE: Shells like ``bash`` and ``zsh`` sometimes attempt to expand
+  wildcards in ways that can be surprising. You may also encounter issues when
+  attempting to copy files whose names contain wildcard characters. For more
+  details about these issues, see "Potentially Surprising Behavior When Using Wildcards"
+  under ``gsutil help wildcards``.
 """
 
 _NAME_CONSTRUCTION_TEXT = """
 <B>HOW NAMES ARE CONSTRUCTED</B>
-  The gsutil cp command strives to name objects in a way consistent with how
-  Linux cp works, which causes names to be constructed in varying ways depending
+  The ``gsutil cp`` command names objects in ways that are consistent with the
+  Linux ``cp`` command. This means that names are constructed depending
   on whether you're performing a recursive directory copy or copying
-  individually named objects; and whether you're copying to an existing or
+  individually-named objects, or whether you're copying to an existing or
   non-existent directory.
 
-  When performing recursive directory copies, object names are constructed that
+  When you perform recursive directory copies, object names are constructed to
   mirror the source directory structure starting at the point of recursive
-  processing. For example, if dir1/dir2 contains the file a/b/c then the
-  command:
+  processing. For example, if ``dir1/dir2`` contains the file ``a/b/c``, then the
+  following command creates the object ``gs://my-bucket/dir2/a/b/c``:
 
     gsutil cp -r dir1/dir2 gs://my-bucket
 
-  will create the object gs://my-bucket/dir2/a/b/c.
-
-  In contrast, copying individually named files will result in objects named by
-  the final path component of the source files. For example, again assuming
-  dir1/dir2 contains a/b/c, the command:
+  In contrast, copying individually-named files results in objects named by
+  the final path component of the source files. For example, assuming again that
+  ``dir1/dir2`` contains ``a/b/c``, the following command creates the object
+  ``gs://my-bucket/c``:
 
     gsutil cp dir1/dir2/** gs://my-bucket
 
-  will create the object gs://my-bucket/c.
-
-  The same rules apply for downloads: recursive copies of buckets and
+  Note that in the above example, the '**' wildcard matches all names
+  anywhere under ``dir``. The wildcard '*' matches names just one level deep. For
+  more details, see ``gsutil help wildcards``.
```

Comment:
Similar to the above comment, I think we want " instead of `` so that a link renders in c.g.c documentation